src/main/java/dk/kb/present/View.java

```diff
@@ -128,9 +136,15 @@ public String apply(DsRecordDto record) {
 
         }
 
+
         for (DSTransformer transformer: this) {
             try {
-                content = transformer.apply(content, metadata);
+                if (transformer.getStylesheet() != null && transformer.getStylesheet().contains("schemaorg2solr.xsl")){
+                    metadata.putIfAbsent("schemaorgjson", content);
+                    content = transformer.apply(placeholderXml, metadata);
```

Comment:
I don't understand why it is necessary to use an empty placeholder here. Why can't the original content be used (and ignored in the XSLT)?

There's probably a very fine and a bit complicated reason for this, so we could take that face2face.

Reply:
That's actually a smart idea! Haven't thought about it

Reply:
Turns out I have thought about it. The content needs to be valid XML and not JSON, therefore the placeholder